content/docs/command-reference/exp/show.md

```diff
@@ -147,109 +119,129 @@ metric or param.
+│ workspace               │ -            │  0.60405 │  0.9608 │ 3000                   │ 2                │ 20170428   │ 100         │ 64              │
+│ master                  │ May 29, 2021 │  0.60405 │  0.9608 │ 3000                   │ 2                │ 20170428   │ 100         │ 64              │
+│ ├── 6b338f8 [exp-3315b] │ 08:03 PM     │  0.58589 │   0.945 │ 2000                   │ 2                │ 20170428   │ 100         │ 64              │
+│ ├── d384680 [exp-bc055] │ 08:03 PM     │  0.51799 │ 0.92333 │ 500                    │ 2                │ 20170428   │ 100         │ 64              │
+│ └── d7fdde2 [exp-1b262] │ 08:03 PM     │  0.56447 │ 0.94713 │ 1500                   │ 2                │ 20170428   │ 100         │ 64              │
+└─────────────────────────┴──────────────┴──────────┴─────────┴────────────────────────┴──────────────────┴────────────┴─────────────┴─────────────────┘
+```
+
+Combine `--only-changed` with `--drop` and/or `--keep`:
+
+```dvc
+$ dvc exp show --only-changed --drop 'neutral:**Created**' --keep 'train.(?!seed)'
 ```
 
 ```dvctable
-┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┓
-┃ neutral:**Experiment**            ┃ neutral:**Created**      ┃     metric:**auc** ┃ param:**featurize.max_features** ┃
-┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━┩
-│ workspace             │ -            │ 0.61314 │ 1500                   │
-│ 10-bigrams-experiment │ Jun 20, 2020 │ 0.61314 │ 1500                   │
-│ ├── exp-e6c97         │ Oct 21, 2020 │ 0.61314 │ 1500                   │
-│ ├── exp-1dad0         │ Oct 09, 2020 │ 0.57756 │ 2000                   │
-│ └── exp-1df77         │ Oct 09, 2020 │ 0.51676 │ 500                    │
-└───────────────────────┴──────────────┴─────────┴────────────────────────┘
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ neutral:**Experiment**              ┃ metric:**avg_prec** ┃ metric:**roc_auc** ┃ param:**featurize.max_features** ┃ param:**train.n_est** ┃ param:**train.min_split** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace               │  0.60405 │  0.9608 │ 3000                   │ 100         │ 64              │
+│ master                  │  0.60405 │  0.9608 │ 3000                   │ 100         │ 64              │
+│ ├── d384680 [exp-bc055] │  0.51799 │ 0.92333 │ 500                    │ 100         │ 64              │
+│ ├── 6b338f8 [exp-3315b] │  0.58589 │   0.945 │ 2000                   │ 100         │ 64              │
+│ └── d7fdde2 [exp-1b262] │  0.56447 │ 0.94713 │ 1500                   │ 100         │ 64              │
+└─────────────────────────┴──────────┴─────────┴────────────────────────┴─────────────┴─────────────────┘
 ```
 
 Sort experiments by the `auc` metric, in ascending order:
```

Comment:
Hm, is it better to specify `ascending order` here if the command doesn't use `--sort-order`? It shows default behavior but doesn't show use of `--sort-order` like it did before.

Reply:
Should I replace with `descending` order ? 

I think showing an option that it's not needed might be counterproductive.

Reply:
Agreed. I'm fine with whatever you want to do here.